src/utils/components/SSHAccess/components/SSHServiceSelect.tsx

```diff
@@ -65,8 +65,15 @@ const SSHServiceSelect: FC<SSHServiceSelectProps> = ({
             'Opens a specific port on all Nodes in the cluster. If the Node is publicly accessible, any traffic sent to this port is forwarded to the Service.',
           )}
           id={SERVICE_TYPES.NODE_PORT}
-          isDisabled={!nodePortEnabled}
+          isAriaDisabled={!nodePortEnabled}
```

Comment:
why changing this?

Reply:
Because with this change, we achieve the expected behavior and look. Using `isDisabled` does not allow us to display the tooltip while option being disabled, unfortunately.